examples/stem_histogram.py

```diff
@@ -0,0 +1,84 @@
+import glob
+import sys
+import os
+from stempy import io, image
+import numpy as np
+from PIL import Image
+import matplotlib.pyplot as plt
+import matplotlib.ticker as ticker
+
+def print_help():
+    print(
+    '''
+    Usage:
+    python3 stem_histogram.py [dataDir -- path to the data directory]
+                              [outDir -- path to the output image directory]
+    ''')
+
+def save_img(stem_image_data, name):
+    min = np.min(stem_image_data)
+    max = np.max(stem_image_data)
+
+    stem_image_data = stem_image_data.reshape((160, 160))
+    stem_image_data = np.interp(stem_image_data, [min, max], [0, 256])
+    stem_image_data = stem_image_data.astype(np.uint8)
+    img = Image.fromarray(stem_image_data)
+    img.save(name)
+
+def main(argv):
+    print_help()
+    dataDir = argv[0] if len(argv) > 0
+    outDir = argv[1] if len(argv) > 1
+
+    # create output directory if it does not exit
+    if not os.path.exists(outDir):
+        os.mkdir(outDir)
+        print('Output directory', outDir , 'created')
+    else:    
```

Comment:
There is some trailing white space in the file as well. I'd recommend getting your editor to automatically delete trailing white space upon saving the file.

Reply:
I forgot to set this on with my intern laptop, sorry about that, just turned it on.